cg/cli/set/family.py

```diff
@@ -0,0 +1,62 @@
+"""Set family attributes in the status database"""
+
+
+import click
+from cg.constants import FAMILY_ACTIONS, PRIORITY_OPTIONS
+
+
+@click.command()
+@click.option("-a", "--action", type=click.Choice(FAMILY_ACTIONS), help="update family action")
+@click.option("-c", "--customer-id", type=click.STRING, help="update customer")
+@click.option("-g", "--panel", "panels", multiple=True, help="update gene panels")
+@click.option("-p", "--priority", type=click.Choice(PRIORITY_OPTIONS), help="update priority")
+@click.argument("family_id")
+@click.pass_context
+def family(
+    context: click.Context,
+    action: str,
+    priority: str,
+    panels: [str],
+    family_id: str,
+    customer_id: str,
+):
+    """Update information about a family."""
+
+    family_obj = context.obj["status"].family(family_id)
+    if family_obj is None:
+        click.echo(click.style(f"Can't find family {family_id}", fg="red"))
```

Comment:
I think all those messages should be changed to LOG messages and all `context.abort()` -> raise click.Abort

Reply:
fixed